extra_data/write_cxi.py

```diff
@@ -21,15 +21,21 @@ class VirtualCXIWriter:
     Parameters
     ----------
 
-    detdata: extra_data.components.MPxDetectorBase
+    detdata: extra_data.components.XtdfDetectorBase
       The detector data interface for the data to gather in this file.
     """
     def __init__(self, detdata):
         self.detdata = detdata
 
         self.modulenos = sorted(detdata.modno_to_source)
 
-        frame_counts = detdata.frame_counts
+        # The following line could be tricky, but there is no pulseId
+        # in Jungfrau data.
+        self.pulse_id_label = 'image.pulseId'
```

Comment:
It's slightly surprising that all of the `_label` variables are single words except this one. :confused:

Reply:
Yes, I was following how it was implemented before, but now I've changed it to the single word in: https://github.com/European-XFEL/EXtra-data/commit/cc73dadb190a2c3e831a1fb08259f2752871ee5d.